### ISSUE TITLE
Please release the training parameters for modelnet_ts_rpmnet_noise

### ISSUE THREAD

**Issue by wzm2256** [hxwork/OMNet_Pytorch#2]
Hi, thanks for releasing the pytorch version code.

Please release the training json file for modelnet_ts_rpmnet_noise settings.

By the way, I wonder that what is the GPU memory requirement for the settings? (batchsize=64).
I use a Nvidia TiTAN with 12G memory, but that is still not sufficient, and my current setting is batchsize=16.

**Comment by hxwork** [comment]
Hi,

1. For the training parameters of `modelnet_ts_rpmnet_noise` data, it should be the same as the parameters described in our paper. You can also find it in `./experiments/experiment_omnet/params.json`.
2. For the training batch size, we set it to 64 in all experiments, and we train our model on 4 RTX 2080Ti. Besides, we also tried to train our model with lower batch sizes, such as 8 or 16, and we found that they lead to worse results.

Hao

**Comment by wzm2256** [comment]
Thanks for your reply. 

1. I am worried that the json file in the experiments folder is not exactly the same as the settings described in the paper. For example, the training epoch is set to 10,000 in the json file. With 4196 training data and batch size 64, the total iteration should be 650K (4196/64*10000), which is much larger than the 260K specified in the paper. In addition, the learning rate schedule factor is set to 0.1 in the paper, but is 1.0 in the json file. Please let me know whether these differences are important.

2. How long does it takes to train the model? Assume it takes 1 minute to train an epoch, then 10,000 epoch will take almost 7 days. Please let me know whether I get this correct.

**Comment by hxwork** [comment]
1. Although we set the total training epoch to 10,000, however, the model with the best performance may appear early before the end of the entire training process. Since the best models of all experiments are chosen independently, we can not ensure all of them appear at the same epoch, so that setting a large training epoch is necessary. Besides, the learning rate is not a very important parameter in our experiments, and you can just set it to 1e-4 without decay or you can multiply it by 0.1 when you find the best performance has not been updated for a long time.
2. As described in 1, the best models of different experiments appear at different epochs, since it depends on the difficulty of the registration data, so that it is hard to say a specific training time. You are correct that finishing the entire training process will take a week with `batch_size=64`.

**Comment by wzm2256** [comment]
Thanks for your reply.
